pgxn/neon/control_plane_connector.c

```diff
@@ -741,6 +741,13 @@ NeonProcessUtility(
 			break;
 		case T_DropdbStmt:
 			HandleDropDb(castNode(DropdbStmt, parseTree));
+                        /*
+                         * We do this here to hack around the fact that Postgres performs the drop
+                         * INSIDE of standard_ProcessUtility, which means that if we try to
+                         * abort the drop normally it'll be too late. DROP DATABASE can't be inside
+                         * of a transaction block anyway, so this should be fine to do.
+                         */
+                        NeonXactCallback(XACT_EVENT_PRE_COMMIT, NULL);
```

Comment:
Can you match the indent? Looks like maybe a tabs vs spaces thing. Code formatting would be useful for the C code. Something more useful than pg_indent though :)

Reply:
Changed it to tabs